app/controllers/concerns/fishrappr/catalog.rb

```diff
@@ -86,12 +86,17 @@ def toggle_highlight
 
   def download_text(document=nil)      
     @response, @document = fetch params[:id]
-
     document = @document unless document
     ocr = document.fetch('page_text')
     send_data ocr, filename: document.id+'.txt'
   end
 
+   def download_issue(document=nil)      
```

Comment:
Can you make this download_issue_text? We'll have a download_issue_pdf method in this class that adds the fishrappr front page.